irmin-pack-utils.opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer:   "Gwenaelle@tarides.com"
+authors:      ["Gwenaëlle Lecat"]
+license:      "ISC"
+homepage:     "https://github.com/mirage/irmin"
+bug-reports:  "https://github.com/mirage/irmin/issues"
+dev-repo:     "git+https://github.com/mirage/irmin.git"
+doc:          "https://mirage.github.io/irmin/"
+
+build: [
+ ["dune" "subst"] {dev}
+ ["dune" "build" "-p" name "-j" jobs]
+ ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+available: arch != "arm32" & arch != "x86_32"
+
+depends: [
+  "ocaml"       {>= "4.01.0"}
+  "dune"        {>= "2.9.0"}
+  "irmin-pack"  {= version}
+  "cmdliner"    {>= "1.1.0"}
+  "irmin-test"  {with-test & = version}
+  "alcotest"    {with-test}
+]
+
+synopsis: "Utils for Irmin-pack"
```

Comment:
Instead of a new package, I'd like to propose renaming `irmin-tezos-utils` to `irmin-pack-tools` and add all of our binaries to it. The current tezos util, could be named something like "tezos_explorer" or made to work with general irmin pack as well (following something like what [irmin_fsck](https://github.com/mirage/irmin/blob/main/test/irmin-tezos/irmin_fsck.ml) does).

Reply:
I'm okay with it, following a discussion on the team slack, we had agreed with @art-w that spliting the packages, but I have nothing against merging them, maybe you have a personal opinion @art-w ?

Reply:
Either way is fine for me! I think we are going to add more small debugging tools to inspect stores, so the name `irmin-pack-tools` is pretty good :)

Reply:
I'll create the package `irmin-pack-tools` but keep the merge from `irmin-tezos-utils` to a later PR